client/components/flag/README.md

```diff
@@ -16,4 +16,3 @@ import Flag from 'components/flag';
    * is-success
    * is-warning
    * is-error
-* `icon` – String for the desired the noticon class (i.e. "noticon-lock").
```

Comment:
Maybe instead of removing this line, it should be moved to an "Optional props" section?